### PR TITLE
update code for upstream notifications plugin

### PR DESCRIPTION
**Pull request by ydayagi**
FLPATH-1249
https://issues.redhat.com/browse/FLPATH-1249

### CODE REVIEW

**Comment by ydayagi** [comment]
converted to draft until orchestrator starts using the upstream notification plugin

File: aap-job/src/main/resources/aap-job.sw.yaml [hunk 1]
```diff
@@ -70,31 +70,29 @@ states:
...
-            topic: "AAP Job Workflow"
-            targetUsers:
-              - "default/guest"
-            targetGroups: []
-            actions:
-              - title: "AAP Job Output"
-                url: .launchedJob.outputUrl
+            recipients:
+              type: entity
+              entityRef:  ["user:default/guest"]
```
Comment (masayag) [inline aap-job/src/main/resources/aap-job.sw.yaml:75]: for the sake of compatibility with other workflows, can we externalize this to be a parameter?

File: aap-job/src/main/resources/aap-job.sw.yaml [hunk 2]
```diff
@@ -70,31 +70,29 @@ states:
...
+            payload:
+              title: '"AAP Job Workflow " + $WORKFLOW.instanceId + " Failed"'
+              description: '"AAP job workflow id: " + $WORKFLOW.instanceId + " for job template id: " + (.jobTemplateId|tostring) + " failed."'
+              scope: "AAP Job Workflow"
```
Comment (masayag) [inline aap-job/src/main/resources/aap-job.sw.yaml:79]: what is the use of `scope`?

File: aap-job/src/main/resources/aap-job.sw.yaml [hunk 3]
```diff
@@ -70,31 +70,29 @@ states:
...
-            topic: "AAP Job Workflow"
-            targetUsers:
-              - "default/guest"
-            targetGroups: []
-            actions:
-              - title: "AAP Job Output"
-                url: .launchedJob.outputUrl
+            recipients:
+              type: entity
+              entityRef:  ["user:default/guest"]
```
Comment (masayag) [inline aap-job/src/main/resources/aap-job.sw.yaml:91]: same

File: aap-job/src/main/resources/application.properties [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # Notifications Settings
 quarkus.rest-client.notifications_openapi_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://localhost:7007/api/notifications}
```
Comment (masayag) [inline aap-job/src/main/resources/application.properties:2]: Does the URL remain the same?

File: escalation-service-now/src/main/resources/serviceNowEscalation.sw.yaml [hunk 5]
```diff
@@ -81,15 +81,15 @@ states:
...
-            origin: "Serverless Workflow Example"
-            targetUsers: "[\"default/manager\"]"
-            topic: "Approval Request"
-            targetGroups: []
-            actions:
-              - title: "Approve"
-                url: " (.sn_url) + \"/nav_to.do?uri=change_request.do?sys_id=\" + (.createdChangeRequest.result.sys_id) "
+            recipients:
+              type: entity
+              entityRef:  ["user:default/manager"]
```
Comment (masayag) [inline escalation-service-now/src/main/resources/serviceNowEscalation.sw.yaml:86]: same here.

File: mta/v6.2.2/src/main/resources/application-dev.properties [hunk 6]
```diff
@@ -6,4 +6,5 @@ quarkus.log.category.\"org.apache.http\".level=DEBUG
 # Added
 quarkus.http.port=8080
 
-quarkus.kogito.devservices.enabled=false
\ No newline at end of file
+quarkus.kogito.devservices.enabled=false
```
Comment (masayag) [inline mta/v6.2.2/src/main/resources/application-dev.properties:9]: pls revert the changes for the MTA. this is being re-worked now as  serverless-workflow to be productized under https://github.com/parodos-dev/serverless-workflows/pull/198
(currently it is closed, but it will be reopened or replaced by a new one).
in any case, once the PR is pushed, the workflows will be deleted from this repository.

File: aap-job/src/main/resources/aap-job.sw.yaml (hunk 1 above)
Comment (ydayagi) [reply aap-job/src/main/resources/aap-job.sw.yaml:75]: this is not due to my changes. but in order to save more PRs and jira issues I can do it here

File: aap-job/src/main/resources/aap-job.sw.yaml (hunk 2 above)
Comment (ydayagi) [reply aap-job/src/main/resources/aap-job.sw.yaml:79]: we had a field 'origin'. there's no real use for it at the moment. the only available field in the new plugin that seemed to fit was 'scope'.

File: aap-job/src/main/resources/application.properties (hunk 4 above)
Comment (ydayagi) [reply aap-job/src/main/resources/application.properties:2]: the base url remains the same

File: mta/v6.2.2/src/main/resources/application-dev.properties (hunk 6 above)
Comment (ydayagi) [reply mta/v6.2.2/src/main/resources/application-dev.properties]: done

File: aap-job/src/main/resources/aap-job.sw.yaml (hunk 1 above)
Comment (ydayagi) [reply aap-job/src/main/resources/aap-job.sw.yaml]: fixed

**Review by masayag — APPROVED** [review]
